ext/reflection/php_reflection.c

```diff
@@ -456,7 +456,6 @@ static void _class_string(string *str, zend_class_entry *ce, zval *obj, char *in
 		zend_class_constant *c;
 
 		ZEND_HASH_FOREACH_STR_KEY_PTR(&ce->constants_table, key, c) {
-			zval_update_constant_ex(&c->value, c->ce);
```

Comment:
It was called twice - first here, and then in the `_class_const_string`. In my understanding there needs to be only one call to resolve the validity of this graph, right?